components/event-sources/reconciler/httpsource/reconcile.go

```diff
@@ -124,7 +126,21 @@ func httpSourceByKey(key string, l sourceslistersv1alpha1.HTTPSourceLister) (*so
 
 // reconcileSink reconciles the state of the HTTP adapter's sink (Channel).
 func (r *Reconciler) reconcileSink(src *sourcesv1alpha1.HTTPSource) (*messagingv1alpha1.Channel, error) {
```

Comment:
The logic of this function is now a little bit more than just "create if not exists" because we now also ensure the Channel's labels are synchronized.